src/main/flight/mixer.c

```diff
@@ -721,12 +722,37 @@ static void applyFlipOverAfterCrashModeToMotors(void)
     }
 }
 
+static float applyThrustLinearization(float motorOutput)
+{
+#ifdef USE_THRUST_LINEARIZATION
+    if (motorConfig()->thrustLinearization) {
+        float tl = motorConfig()->thrustLinearization * 0.01f;
```

Comment:
this code is on critical path, ideally parameters would be set in some Init function.

Reply:
I've moved this into pid.c and now initialize the parameters once.